cmd/kubepkg/templates/latest/rpm/kubelet/kubelet.spec

```diff
@@ -17,7 +17,6 @@ Requires: ethtool
 Requires: iproute
 Requires: ebtables
 Requires: conntrack
-Obsoletes: kubernetes-cni
 Conflicts: kubernetes-cni
```

Comment:
In either case you won't be able to install both the new kubelet (with the cni binaries) and the cni package that comes with CRI-O.
You would see something similar to https://github.com/kubernetes/release/pull/1367#issuecomment-646069479

Reply:
Thank you for the clarification, let’s move that issue to another PR.

Reply:
Sorry to keep going back to this, but short of CIO-O and any other packages out there that already rely on the kubernetes-cni RPM, having to change their RPM spec, and remove the dependency on that package I don't see a solution here.
With this additional information, should we reconsider moving the cni binaries out of the kubelet spec and making it its own package?

Reply:
Hm I see you lint. OTOH I think that this would be a move backwards from my point of view (see the linked issues in https://github.com/kubernetes/release/pull/1309).

Would it be possible to use an almost unique location for the plugins (rather than owning binaries in `/opt`)?